src/themes/Treedip/SoftClose/SoftClose.scss

```diff
@@ -12,7 +12,12 @@
 		&-columns {
 			padding: 8px;
 			flex-wrap: nowrap;
-			justify-content: center;
+			justify-content: space-between;
+
+			.wmde-banner-close {
+				position: static;
+				margin-right: -40px;
+			}
```

Comment:
This is now unnecessary with the change to `ButtonCloseWithText` which fixes the "components with same name" issue (by adding a double class selector).